### ISSUE TITLE
typing.Literal isn't supported on Python 3.9.0

### ISSUE THREAD

**Issue by fabiopedrosa** [Fatal1ty/openapify#3]
I'm getting a exception running the sample code in README:

`mashumaro.exceptions.UnserializableDataError: typing.Literal[<SecuritySchemeType.API_KEY: 'apiKey'>] as a field type is not supported by mashumaro`

**Comment by Fatal1ty** [comment]
Hi @fabiopedrosa 

Could you tell me which version of `openapify` and `mashumaro` you have installed?

**Comment by fabiopedrosa** [comment]
@Fatal1ty the latest, 0.3.8 for openapify and 3.7 for mashumaro

```
Traceback (most recent call last):
  File "...users_lambda.py", line 10, in <module>
    from openapify.core.models import RouteDef
  File "C:\Program Files\Python39\lib\site-packages\openapify\__init__.py", line 1, in <module>
    from .core.builder import build_spec
  File "C:\Program Files\Python39\lib\site-packages\openapify\core\builder.py", line 24, in <module>
    from openapify.core.document import OpenAPIDocument
  File "C:\Program Files\Python39\lib\site-packages\openapify\core\document.py", line 10, in <module>
    from openapify.core.openapi.models import SecurityScheme, Server
  File "C:\Program Files\Python39\lib\site-packages\openapify\core\openapi\models.py", line 143, in <module>
    class APIKeySecurityScheme(SecurityScheme):
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\mixins\dict.py", line 24, in __init_subclass__
    compile_mixin_unpacker(cls, **builder_params["unpacker"])
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\core\meta\mixin.py", line 49, in compile_mixin_unpacker
    builder.add_unpack_method()
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\core\meta\code\builder.py", line 398, in add_unpack_method
    self._add_unpack_method_lines(method_name)
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\core\meta\code\builder.py", line 313, in _add_unpack_method_lines
    self._unpack_method_set_value(
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\core\meta\code\builder.py", line 440, in _unpack_method_set_value
    unpacked_value = UnpackerRegistry.get(
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\core\meta\types\common.py", line 92, in get
    expr = packer(spec)
  File "C:\Program Files\Python39\lib\site-packages\mashumaro\core\meta\types\unpack.py", line 448, in unpack_special_typing_primitive
    raise UnserializableDataError(
mashumaro.exceptions.UnserializableDataError: typing.Literal[<SecuritySchemeType.API_KEY: 'apiKey'>] as a field type is not supported by mashumaro
```

**Comment by Fatal1ty** [comment]
Hmm, this is strange. I run this snippet without errors on Python 3.9.16:
```python
from openapify.core.models import RouteDef
from openapify import build_spec
from openapify import security_requirements
from openapify.core.openapi.models import (
    APIKeySecurityScheme,
    SecuritySchemeAPIKeyLocation,
)

XAuthTokenSecurityRequirement = {
    "x-auth-token": APIKeySecurityScheme(
        name="X-Auh-Token",
        location=SecuritySchemeAPIKeyLocation.HEADER,
    )
}

@security_requirements(XAuthTokenSecurityRequirement)
def secure_operation():
    ...

spec = build_spec([RouteDef("/secure_path", "get", secure_operation)])
print(spec.to_yaml())
```

Can you give me more info? Exact Python version and the minimal reproducible code would be helpful.

**Comment by fabiopedrosa** [comment]
@Fatal1ty Upgrading python did fix it. I was on 3.9.0

**Comment by Fatal1ty** [comment]
Thank you for feedback. This bug will be fixed in mashumaro:
* https://github.com/Fatal1ty/mashumaro/issues/103

I'll close this issue after updating the dependency.

**Comment by Fatal1ty** [comment]
I just released [mashumaro 3.8](https://github.com/Fatal1ty/mashumaro/releases/tag/v3.8) with the fix.
